app-name/src/Entries.jsx

```diff
@@ -0,0 +1,34 @@
+import RenderEntry from './RenderEntry';
+import { data } from './data';
+
+export default function Entries({ onClick }) {
+  return (
+    <div data-view="entries" className="entries">
+      <div className="row">
+        <div className="column-full">
+          <div className="new-header">
+            <h2>Entries</h2>
+            <a
+              data-view="entry-form"
+              className="new-button"
+              id="entry-form-anchor"
+              onClick={() => onClick('create')}>
```

Comment:
This works, but it means that the Entries component has to know exactly how App decides which component to render. I think it would make the code easier to maintain and work with if you named the prop `onCreate` and didn't pass it anything -- let App decide what to do and how to switch the view.